src/app/app.module.ts

```diff
@@ -304,9 +304,7 @@ export const sunbirdSdkFactory =
           dbName: 'GenieServices.db'
         },
         deviceRegisterConfig: {
-          host: buildConfigValues['DEVICE_REGISTER_BASE_URL'],
-          apiPath: '/v3/device',
-          deviceProfileApiPath: '/api/v3/device',
+          apiPath: '/api/v3/device'
```

Comment:
DEVICE_REGISTER_BASE_URL
delete this unused key from build-extras.gradle and sunbird.properties.example file.

Reply:
Removed said configuration